### PR TITLE
Document more parameters

### PR DESCRIPTION
**Pull request by RemiLehe**
This PR updates the documentation and describes more of the input parameters.

### CODE REVIEW

File: Docs/source/building/building.rst [hunk 1]
```diff
@@ -44,7 +44,7 @@ This will generate an executable file in the ``Bin`` directory.
         * Choose a given compiler
 
     For a description of these different options, see the `corresponding page
-    <https://amrex-codes.github.io/amrex/BuildingAMReX.html#building-with-gnu-make>`__ in
+    <https://amrex-codes.github.io/amrex/docs_html/BuildingAMReX.html>`__ in
```
Comment (RemiLehe) [inline Docs/source/building/building.rst]: The link was broken.

**Comment by RemiLehe** [comment]
Good point. I'll add a link to the openPMD-viewer.

**Comment by RemiLehe** [comment]
I added more info on openPMD/plotfiles and how to visualize them (including a small page on openPMD-viewer). }

@MaxThevenet do you think that this could be merged?

**Comment by RemiLehe** [comment]
Actually: my mistake, I made those changes directly on `dev` ; sorry!

**Comment by MaxThevenet** [comment]
`thumbs-up`

**Pull request merged**